tests/14-policy-enforcement-docker.sh

```diff
@@ -64,15 +68,15 @@ function check_endpoints_policy_enabled {
 #######################################
 function check_endpoints_policy_disabled {
   local NUM_EPS=$1
-  echo "---- checking if ${NUM_EPS} endpoints have policy enforcement disabled ----"
+  log "checking if ${NUM_EPS} endpoints have policy enforcement disabled"
   cilium endpoint list 
   POLICY_DISABLED_COUNT=`eval ${DISABLED_CMD}`
   if [ "${POLICY_DISABLED_COUNT}" -ne "${NUM_EPS}" ] ; then 
     cilium config
     cilium endpoint list
     abort "Policy Enforcement  should be set to 'Disabled' since policy enforcement was set to never be enabled"
   fi
-  echo  "---- ${NUM_EPS} endpoints have policy enforcement disabled; continuing ----"
+  log  "${NUM_EPS} endpoints have policy enforcement disabled; continuing"
```

Comment:
`s/log  "${NUM_EPS}"/log "${NUM_EPS}"`

Reply:
What is the difference between what is being substituted vs. what is substituted here? I can't discern a difference.

Reply:
A small space between `log` and `"${NUM_EPS}"` It's a really small nit :)